npm plugin overwrite every scoped packages as public

**Describe the bug**

Npm publish of private scoped packages override the npm setting and turn the package public. 

**To Reproduce**

Have a scoped package, release it, it will be public.
Change the npm package setting to `private` manually, release again, the package will become public again.

**Expected behavior**

Scoped packages should be published as private.

**Screenshots**

**Desktop (please complete the following information):**

irrelevant

**Additional context**

The alternative of setting `"private": true` on the package.json is not a solution since npm will refuse to publish it.
```
npm ERR! code EPRIVATE
npm ERR! This package has been marked as private
npm ERR! Remove the 'private' field from the package.json to publish it.
```

--- a/plugins/npm/__tests__/npm.test.ts
+++ b/plugins/npm/__tests__/npm.test.ts
@@ -581,27 +581,6 @@ describe('publish', () => {
     ]);
   });
 
-  test('should publish public scoped packages to public', async () => {
-    const plugin = new NPMPlugin();
-    const hooks = makeHooks();
-
-    plugin.apply({
-      config: { prereleaseBranches: ['next'] },
-      hooks,
-      baseBranch: 'master',
-      logger: dummyLog()
-    } as Auto.Auto);
-
-    readResult = `
-      {
-        "name": "@scope/test"
-      }
-    `;
-
-    await hooks.publish.promise(Auto.SEMVER.patch);
-    expect(exec).toHaveBeenCalledWith('npm', ['publish', '--access', 'public']);
-  });
-
   test('should publish private scoped packages to private', async () => {
     const plugin = new NPMPlugin();
     const hooks = makeHooks();
@@ -655,32 +634,6 @@ describe('canary', () => {
     await hooks.canary.promise(Auto.SEMVER.patch, '.123.1');
     expect(exec.mock.calls[0]).toContain('npm');
     expect(exec.mock.calls[0][1]).toContain('1.2.4-canary.123.1.0');
-  });
-
-  test('publishes to public for scoped packages', async () => {
-    const plugin = new NPMPlugin();
-    const hooks = makeHooks();
-
-    plugin.apply(({
-      config: { prereleaseBranches: ['next'] },
-      hooks,
-      baseBranch: 'master',
-      logger: dummyLog(),
-      getCurrentVersion: () => '1.2.3',
-      git: {
-        getLatestRelease: () => '1.2.3',
-        getLatestTagInBranch: () => '1.2.3'
-      }
-    } as unknown) as Auto.Auto);
-
-    readResult = `
-      {
-        "name": "@scope/test"
-      }
-    `;
-
-    await hooks.canary.promise(Auto.SEMVER.patch, '');
-    expect(exec.mock.calls[1][1]).toContain('public');
   });
 
   test('use lerna for monorepo package', async () => {

--- a/plugins/npm/src/index.ts
+++ b/plugins/npm/src/index.ts
@@ -624,8 +624,6 @@ export default class NPMPlugin implements IPlugin {
       }
 
       auto.logger.verbose.info('Detected single npm package');
-      const { private: isPrivate, name } = await loadPackageJson();
-      const isScopedPackage = name.match(/@\S+\/\S+/);
       const current = await auto.getCurrentVersion(lastRelease);
       const canaryVersion = determineNextVersion(
         lastRelease,
@@ -644,13 +642,9 @@ export default class NPMPlugin implements IPlugin {
         '--no-git-tag-version',
         ...verboseArgs
       ]);
+
       const publishArgs = ['--tag', 'canary'];
-      await execPromise(
-        'npm',
-        !isPrivate && isScopedPackage
-          ? ['publish', '--access', 'public', ...publishArgs, ...verboseArgs]
-          : ['publish', ...publishArgs, ...verboseArgs]
-      );
+      await execPromise('npm', ['publish', ...publishArgs, ...verboseArgs]);
 
       if (this.canaryScope) {
         await gitReset();
@@ -800,15 +794,7 @@ export default class NPMPlugin implements IPlugin {
           ...verboseArgs
         ]);
       } else {
-        const { private: isPrivate, name } = await loadPackageJson();
-        const isScopedPackage = name.match(/@\S+\/\S+/);
-
-        await execPromise(
-          'npm',
-          !isPrivate && isScopedPackage
-            ? ['publish', '--access', 'public', ...tag, ...verboseArgs]
-            : ['publish', ...tag, ...verboseArgs]
-        );
+        await execPromise('npm', ['publish', ...tag, ...verboseArgs]);
       }
 
       await execPromise('git', [

